Automattic/wp-calypso
# Remove Publish Confirmation feature flag

This PR removes the feature flag for Post Publish Confirmation (`post-editor/delta-post-publish-flow`) but leaves the `postPublishConfirmation` AB test in place.

This is part of a larger epic (See p3Ex-2ri-p2).

To test:

- Check out branch.
- Run `npm start`
- Create a new post.
- Once the editor is loaded, a variation for the `postPublishConfirmation` should be set (`localStorage.setItem('debug', 'calypso:abtests');`).
- Publish the new post.
  - `showPublishConfirmation`: A confirmation sidebar should be shown on Publish.
  - `noShowPublishConfirmation`: No confirmation sidebar is shown on Publish.
- Create a new post.
- Use the dev ABTest tool to set the other variation (or `localStorage.setItem('ABTests','{"postPublishConfirmation_20170801":"showPublishConfirmation"}')` or use `noPublishConfirmation` for the other variation).
- Verify that it works correctly.
